src/immutable/Colony.ts

```diff
@@ -40,13 +31,13 @@ export type ColonyType = $ReadOnly<
 export type ColonyProps<T extends keyof ColonyType> = Pick<ColonyType, T>;
 
 type ColonyRecordProps = Shared & {
-  tokens?: ImmutableMapType<Address, TokenReferenceRecordType>;
+  tokens?: ImmutableMap<Address, TokenReferenceRecord>;
 };
 
-const defaultValues: ColonyRecordProps = {
+const defaultValues: DefaultValues<ColonyRecordProps> = {
   avatarHash: undefined,
-  canMintNativeToken: undefined,
-  canUnlockNativeToken: undefined,
+  canMintNativeToken: false,
```

Comment:
This change seems to prevent me from being able to mint new tokens in a colony (in `TabContribute.tsx`)